node/node.d.ts

```diff
@@ -1418,6 +1418,7 @@ declare module "fs" {
     export interface WriteStream extends stream.Writable {
         close(): void;
         bytesWritten: number;
+        path: string;
```

Comment:
According to the docs, this can be either a string or a Buffer depending on what's passed to the `createWriteStream` function.